vitrina/datasets/urls.py

```diff
@@ -58,6 +59,11 @@
         {'tag_model': Dataset.tags.tag_model},
         name='autocomplete_tags',
     ),
+    path(
+        'dataset/stats/supervisor/',
```

Comment:
```suggestion
        'dataset/stats/jurisdiction/',
```

I thing `supervisor` is about human roles, `jurisdiction` about organization relations.